app/screens/manage_channel_members/manage_channel_members.tsx

```diff
@@ -68,47 +75,29 @@ export default function ManageChannelMembers({
     currentTeamId,
     currentUserId,
     tutorialWatched,
+    teammateDisplayNameSetting,
 }: Props) {
     const serverUrl = useServerUrl();
     const theme = useTheme();
-    const {formatMessage} = useIntl();
+    const {formatMessage, locale} = useIntl();
 
     const searchTimeoutId = useRef<NodeJS.Timeout | null>(null);
     const mounted = useRef(false);
 
     const [isManageMode, setIsManageMode] = useState(false);
     const [profiles, setProfiles] = useState<UserProfile[]>(EMPTY);
+    const hasMoreProfiles = useRef(false);
```

Comment:
We only want to search on the server, if we don't have all the profiles loaded. This is set later.